fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -854,6 +854,47 @@ private void checkAssignedTargetIndexName(String baseIndexName, String targetInd
         }
     }
 
+    private boolean checkPartitionColumnChangeAllowed(Column oldPartitionColumn, Column newPartitionColumn) {
```

Comment:
We are better to describe the problem clearly in the commit message. 
Don't make me guess what's the problem.